pymc/tests/test_ode.py

```diff
@@ -209,44 +228,85 @@ def system_1(y, t, p):
 class TestErrors:
     """Test running model for a scalar ODE with 1 parameter"""
 
+    @staticmethod
     def system(y, t, p):
         return np.exp(-t) - p[0] * y[0]
 
     times = np.arange(0, 9)
 
-    ode_model = DifferentialEquation(func=system, t0=0, times=times, n_states=1, n_theta=1)
+    ode_model = DifferentialEquation(func=system.__func__, t0=0, times=times, n_states=1, n_theta=1)
```

Comment:
This would be typical for a `setup_method`:

```python
def setup_method(self, method):
    def system(y, t, p):
        return np.exp(-t) - p[0] * y[0]
    self.times = np.arange(0, 9)
    self.ode_model = DifferentialEquation(func=system, t0=0, times=self.times, n_states=1, n_theta=1)
```
Also see https://docs.pytest.org/en/latest/how-to/xunit_setup.html

Reply:
I implemented the setup method and I will try to rebase onto main next.

Reply:
Looks like the rebase fixed the CI pipeline. Let me know what you think.